src/js/Templates/NonMedia/NonMediaBody.js

```diff
@@ -18,14 +18,14 @@ export default class NonMediaBody extends React.Component {
         if(textFields.length > 2) {
             for(i=0; i<textFields.length; i++) {
                 if(i < 2) {
-                    textFields[i].className = "t-large t-light th-f-color non-media-text-4"; 
+                    textFields[i].className = "t-large t-light th-f-color non-media-text-4 non-media-oneline";
                 } else {
                     textFields[i].className = "t-small t-light th-f-color-secondary non-media-text-4"; 
```

Comment:
Probably should also add the overflow changes for mainField3 and mainField4
![Screenshot from 2021-08-18 14-41-50](https://user-images.githubusercontent.com/10703941/129954388-649b6e5f-5982-4eb4-95a3-516de8636535.png)

Reply:
Good catch, fixed with the `word-break: break-all;` property in https://github.com/smartdevicelink/generic_hmi/pull/425/commits/967fce9aa745ad906c444b53862a68de124ff383

Reply:
Nice, that seems to have fixed the issue